pandas/tests/series/indexing/test_where.py

```diff
@@ -140,6 +140,9 @@ def test_where():
     rs = s2.where(cond[:3], -s2)
     tm.assert_series_equal(rs, expected)
 
+    with tm.assert_produces_warning(FutureWarning):
+        s.where(s > 1, 10, False)
```

Comment:
Let's make a new test case for this, otherwise existing ones get longer and longer